cargo-temp-bindings/Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "cargo-temp-bindings"
-version = "0.0.0"
+version = "0.0.1"
```

Comment:
@cecton, as I understand it shouldn't be changed, right? It's an internal package which exists only to be populated by Windows bindings.

Reply:
Unfortunately it needs to be published because it's still a dependency. So the version needs to be maintained like the rest.

(It's 0.10 though, not 0.0.1 XD but thats not very important)